go.mod

```diff
@@ -1,18 +1,19 @@
-module scai-gen
+module github.com/in-toto/scai-demos/scai-gen
```

Comment:
Module needs a rename.

Reply:
why? I still only want this to point to the scai-gen subdirectory.

Reply:
And/or at least signal that the module comprises the `scai-gen` pieces, not the demos as a whole.